### DISCUSSION TITLE
who are atozed software?

### DISCUSSION THREAD

**Discussion by AsertCreator** [CosmosOS/Cosmos#2614] [Q&A]
i found this on gocosmos website. apparently, they are sponsors of cosmos and its related projects, but what exactly is this? is this a real company? is it non-profit? how cosmos get sponsored? who are atozed software?

**Comment by 9xbt** [comment]
“Atozed Software is a corporate sponsor of open source projects and provides web space, server resources, and significant development resources in support of these projects.” Found that on Google
